src/ORM/Table.php

```diff
@@ -121,6 +121,13 @@ class Table implements RepositoryInterface, EventListenerInterface
     use EventManagerTrait;
 
     /**
+     * Name of default validation set.
+     *
+     * @var string
+     */
+    const VALIDATOR_DEFAULT = 'default';
```

Comment:
I think in the above the semantic grouping is clearer when the consistent part is the first one.

    VALIDATOR_DEFAULT
    VALIDATOR_OTHER

etc

But with DEFAULT_HOST and DEFAULT_PORT etc there is usually only one value, just not written as magic numbers (because that's bad coding style). So in those cases the order can be more "natural".